Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:20-alpine AS base
+
+# Stage 1: Install dependencies
+FROM base AS installer
+
+WORKDIR /usr/src/app
+RUN apk add --no-cache libc6-compat
+COPY package.json package-lock.json ./
+COPY prisma ./prisma
+
+RUN \
+    if [ -f package-lock.json ]; then \
+    npm ci --ignore-scripts; \
+    else \
+    echo "package-lock.json not found" && exit 1; \
+    fi
+
+# Stage 2: Build stage
+FROM base AS builder
+WORKDIR /usr/src/app
+
+COPY --from=installer /usr/src/app/node_modules ./node_modules
+COPY --from=installer /usr/src/app/prisma ./prisma
```

Comment:
replace this line 23 with `copy . .`  which will copy all file along with prisma in this stage

Reply:
But will it not be better to copy only prisma file?

Reply:
it is copying prisma file from previous stage which doesnot uses prisma file. so instead we can directly copy all file along with prisma file in this stage.